README.md

```diff
@@ -1,106 +1,3 @@
 # Virtual Scroll for Angular Material Table
 
-An Angular Directive, which allow to use [virtual scrolling](https://material.angular.io/cdk/scrolling) in [mat-table](https://material.angular.io/components/table)
```

Comment:
i wouldn't remove the whole readme, but just add the 'Origin Library was forked...' as first sentence here? and keep the rest of the original readme?